sdk/aqueduct/utils/serialization.py

```diff
@@ -272,7 +272,7 @@ def _write_bytes_output(output: bytes) -> bytes:
 
 
 def _write_pickle_output(output: Any) -> bytes:
-    return bytes(pickle.dumps(output))
+    return bytes(pickle.dumps(output, protocol=pickle.DEFAULT_PROTOCOL))
```

Comment:
Can you add a comment here explaining the protocol is for compatibility with 3.7 and also a bit about what it does?